setup.py

```diff
@@ -24,8 +24,8 @@ def get_description():
 setup(
     name="parliament",
```

Comment:
We should probably aim to get a new name decided on before merging this, so we don't interfere with the existing PyPI Parliament library.  Any suggestions from anyone?

Reply:
fine with parliamentarian.